docs/guides/postgres/remote-replica/migration.md

```
@@ -0,0 +1,415 @@
+
+**Distribution / libc match** — the source and the KubeDB image must be built against the
+same libc. Compare `version()` on both sides; the platform triple must match (here:
+`x86_64-pc-linux-musl` on both). If they differ, stop and pick a matching `PostgresVersion`
+— a mismatch is not supported:
+
+```bash
+kubectl exec -n <source-ns> <source-pod> -- psql -U <user> -d postgres -tAc "SELECT version();"
+kubectl exec -n demo pg-mig-0 -c postgres -- psql -U migrator -d postgres -tAc "SELECT version();"
+```
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)